deploy/charts/alluxio-operator/crds/k8s-operator.alluxio.com_alluxioclusters.yaml

```diff
@@ -84,6 +84,16 @@ spec:
                 properties:
                   auth:
                     properties:
+                      client:
+                        properties:
+                          enableAuthentication:
+                            type: boolean
+                        type: object
+                      peer:
+                        properties:
+                          enableAuthentication:
+                            type: boolean
```

Comment:
why do we need to add this? coz in bitnami setting, if peer enableAuthentication is enabled, it will require ca files,
these 2 have to come in pair, and when enabling this, i'm assuming user always wants to use their own ca files.
value: "/opt/bitnami/etcd/certs/peer/{{ .Values.auth.peer.caFilename | default "ca.crt" }}

Reply:
Removed peer authentication